query_processor/services/content_router.py

```
@@ -0,0 +1,113 @@
+                request_type=request_type,
+                **validation_result,
+                status='processing'
+            )
+            
+            # Route to appropriate processor
+            if request_type in ['video', 'playlist']:
+                return await ContentRouterService._route_video_request(query_request)
+            elif request_type == 'topic':
+                return await ContentRouterService._route_topic_request(query_request)
+            else:
+                raise ValueError(f"Unsupported request type: {request_type}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)